container/shared/pom.xml

```diff
@@ -68,25 +72,11 @@
                         <feature>eventadmin</feature>
                     </startupFeatures>
                     <bootFeatures>
-                        <feature>wrap</feature>
```

Comment:
Are all these features part of karaf standard features ?

Reply:
Yup! It was silly to list them manually, it was literally the exact same list in the exact same order.